### ISSUE TITLE
Metabase fails to launch in Chinese

### ISSUE THREAD

**Issue by Dog-Lee** [metabase/metabase#8562]
```
java -jar metabase.jar 
```

and i get the error
```
09-20 11:00:40 INFO metabase.util :: 正在加载Metabase……
09-20 11:00:42 INFO util.encryption :: 已存储的证书加密方式已经为这个Metabase事例启用 🔐 
For more information, see https://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest
Exception in thread "main" java.lang.ExceptionInInitializerError
....
Caused by: java.lang.IllegalArgumentException: can't parse argument number: Login
	at java.text.MessageFormat.makeFormat(MessageFormat.java:1429)
	at java.text.MessageFormat.applyPattern(MessageFormat.java:479)
	at java.text.MessageFormat.<init>(MessageFormat.java:380)
	at puppetlabs.i18n.core$fmt.invoke(core.clj:250)
	at puppetlabs.i18n.core$translate.doInvoke(core.clj:257)
	at clojure.lang.RestFn.invoke(RestFn.java:445)
	at metabase.integrations.ldap$fn__48754.invokeStatic(ldap.clj:48)
	at metabase.integrations.ldap$fn__48754.invoke(ldap.clj:47)
	at metabase.integrations.ldap__init.load(Unknown Source)
	at metabase.integrations.ldap__init.<clinit>(Unknown Source)
	... 126 more
Caused by: java.lang.NumberFormatException: For input string: "Login"
```

what should i do?

**Comment by Dog-Lee** [comment]
it seem about MB_ENCRYPTION_SECRET_KEY.

**Comment by halee830** [comment]
there is a same error with win7 jvm.
what should i do?

**Comment by KidYoungChang** [comment]
There is  a same error with win10 64bit

**Comment by jornh** [comment]
I can reproduce the error in with v0.30.3 by making Java believe I'm from China, and understand Chinese.

    java -Duser.country=CN -Duser.language=zh -jar metabase.jar

=> 💣 

A short term workaround (until a proper fix can be made) for you guys is to set the language to English:

    java -Duser.country=CN -Duser.language=en -jar metabase.jar

Or ... apparently go back to 0.30.2 as noted in http://discourse.metabase.com/t/java-error-and-parameter-date-mismatch/4456:

> The first thing is, when I was trying to update the jar file to version 030.03 on windows 7 (x64 version), the message showed “A java Exception has occurred”, so I have to stick to version 030.02.

**Comment by jornh** [comment]
See PR https://github.com/metabase/metabase/pull/8569 for a proposed fix.

Would be good if one of you @Dog-Lee, @halee830 or @KidYoungChang can verify it.

Also the translation in POeditor is really the proper place to fix this. Feel free to do that - so Metabase can fully wave its 🇨🇳 flag 🙂

**Comment by jornh** [comment]
@Dog-Lee thank you for testing and confirmation of the PR! 👍 

@KidYoungChang if you want to use the workaround with I guess it depends on which Java Service Wrapper you’re using. Seems there are several. At least a few of them looks like they take a config file: http://yajsw.sourceforge.net/#mozTocId210483

The other ways are to build from source with the `release-0.30.4` branch which now has the fix or wait for next release.

**Comment by KidYoungChang** [comment]
> @Dog-Lee thank you for testing and confirmation of the PR! 👍
> 
> @KidYoungChang if you want to use the workaround with I guess it depends on which Java Service Wrapper you’re using. Seems there are several. At least a few of them looks like they take a config file: http://yajsw.sourceforge.net/#mozTocId210483
> 
> The other ways are to build from source with the `release-0.30.4` branch which now has the fix or wait for next release.

thanks for your reply!

**Comment by salsakran** [comment]
fixed in 8569 and going out in 0.30.4 nowish
